etc/config/hook.defaults.properties

```diff
@@ -0,0 +1,12 @@
+# Default settings for Hook
```

Comment:
a `hook.amazon.properties` is also needed for the language to show up in the live site

Reply:
Ok.

Reply:
Oops! I don't know what to put inside this `hook.amazon.properties` file.

Reply:
It will mostly be the same as in the defaults version, but with a few key differences:

- The compiler exe path will be in the form `/opt/compiler-explorer/hook-<version>/...`, whatever the infra repo installs it at.
- The compiler ids need to be unique and never be removed once live, so the ids will also have the form of `hook<versionumber>`

Reply:
Ok.